rpc/src/v1/tests/eth.rs

```diff
@@ -257,13 +257,13 @@ fn eth_get_block() {
 
 #[test]
 fn eth_get_block_by_hash() {
```

Comment:
The json of this test is totally different from the previous one, so the test needs to be rewritten